### PR TITLE
TUSD offboarding plan - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      'post-TUSD-Payload-activation',
+      AaveV2Ethereum.POOL
+    );
+
+    // check it's not bricked
+    ReserveConfig memory configTUSDAfter = _findReserveConfigBySymbol(allConfigsAfter, TUSD_SYMBOL);
+    _withdraw(
+      configTUSDAfter,
+      AaveV2Ethereum.POOL,
+      0x9FCc67D7DB763787BB1c7f3bC7f34d3C548c19Fe,
```
Comment (Zer0dot) [inline src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol:70]: Could probably be pulled to a constant at the top, but not a huge deal regardless

**Review by reem — COMMENTED** [review]
Found a few inconsistencies between the proposal description text and the implementation

**Comment by marczeller** [comment]
Thx for feed back, Pushed @reem good catch, was using wrong interestRate strategy, updated to the right one. @Zer0dot while the rate strategy address is now for some reason diff reports don't show the right one.

File: src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol (hunk 1 above)
Comment (marczeller) [reply src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol:65]: removed const and calling from lib.

File: src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+    uint256 TUSDBalanceAfter = IERC20(AaveV2EthereumAssets.TUSD_UNDERLYING).balanceOf(
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+    assertApproxEqAbs(aTUSDBalanceAfter, 0, 1500 ether, 'aTUSD_LEFTOVER');
+    assertEq(TUSDBalanceAfter, aTUSDBalanceBefore + TUSDBalanceBefore);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'post-TUSD-Payload-activation',
```
Comment (brotherlymite) [inline src/AaveV2_Eth_TUSDOffboardingPlan_20233107/AaveV2_Eth_TUSDOffboardingPlan_20233107.t.sol:56]:
```suggestion
      'postAaveV2_Eth_TUSDOffboardingPlan_20233107',
```
This should fix the diffs.

**Review by rustboyar — APPROVED** [review]

**Pull request merged**